src/models/User.ts

```diff
@@ -1,14 +1,29 @@
 import { Schema, model, models } from 'mongoose'
 import type { User } from 'next-auth'
 
+enum Role {
+  'Applicant',
+  'Hacker',
+  'Exec',
+  'Sponsor',
+}
+
 export interface IUser extends User {
```

Comment:
do we need a field for "user has filled out profile info"?

Reply:
yes we would need something like that to track where they're at